contracts/test-utils/src/web3_wrapper.ts

```diff
@@ -46,7 +46,7 @@ const gethConfigs = {
 };
 const ganacheConfigs = {
     shouldUseInProcessGanache: true,
-    shouldAllowUnlimitedContractSize: env.parseBoolean(EnvVars.UnlimitedContractSize),
+    shouldAllowUnlimitedContractSize: true,
```

Comment:
Nice! With this in mind, we should probably make a task for writing tests similar to `contracts/exchange/test/codesize.ts`.